doc/nrf/device_guides/working_with_nrf/nrf70/developing/raw_tx_operation.rst

```diff
@@ -0,0 +1,128 @@
+     - wifi mode -i1 -st
+     - configure interface instance 1 to Station + TX injector mode
+   * - net_mgmt(NET_REQUEST_WIFI_MODE)
+     - wifi mode -i<interface instance> <configuration>
+     - wifi mode -i1 -s
+     - configure interface instance 1 to pure station mode
+   * - net_mgmt(NET_REQUEST_WIFI_CHANNEL)
+     - wifi channel -i<interface instance> <channel number to set>
+     - wifi channel -i1 -c6
+     - set raw transmission channel to 6 for interface 1 to be used in Station idle mode
+
+Raw packet transmit header
+**************************
+
+The raw packet to be transmitted by an application should be a fully conformant 802.11 packet. In addtion, it should provide transmit parameters that inform the nrf70 series Wi-Fi device driver on how the packet should be transmitted. It should also provide a packet marker identifying itself as a special packet which should be handled differently by the nrf70 series Wi-Fi driver. The transmit parameters and the packet marker together forms the raw packet transmit header. The raw packet transmit header is prepended to the 802.11 conformant raw packet before transmitting the packet to the lower layers by the application.
+
+The raw packet transmit header elements and their description is provided below:
+
+.. list-table:: Wi-Fi raw packet transmission header elements
+   :header-rows: 1
+
+   * - Header elements
+     - Description
+   * - magic_num
+     - magic number to identify a raw packet. It is set to 0x12345678.
+   * - data_rate
+     - Data rate at which packet is to be transmitted. Depends on ``tx_mode`` parameter. If ``tx_mode`` is set to legacy mode, data rate is the number provided. If ``tx_mode`` is set to HT/VHT/HE mode, the data rate is the MCS rate.
+   * - packet_length
+     - Packet length of 802.11 raw packet excluding the raw transmit packet header length.
+   * - tx_mode
+     - Mode describing if packet is VHT, HT, HE or Legacy.
+   * - queue
+     - Wi-Fi access category mapping for packet.
+   * - raw_tx_flag
+     - Flag indicating raw packet transmission. This is reserved for driver use.
+
+Refer structure ``nrfxlib/nrf_wifi/fw_if/umac_if/inc/default/fmac_structs.h/raw_tx_pkt_header`` for the raw packet header.
+
+Refer structure ``nrfxlib/nrf_wifi/fw_if/umac_if/inc/default/fmac_structs.h/nrf_wifi_fmac_ac`` for setting the ``queue`` parameter in the raw packet transmit header.
+
+Refer structure ``nrfxlib/nrf_wifi/fw_if/umac_if/inc/default/fmac_structs.h/nrf_wifi_fmac_rawtx_mode`` for setting the ``tx_mode``  parameter in the raw packet transmit header.
+
+.. _ug_nrf70_developing_raw_packet_transmit_operation:
+
+Raw packet transmit operation
+*****************************
+
+The raw packet data has to be encompassed in an 802.11 packet and prepended with the raw transmit header before it is transmitted to the nrf70 Wi-Fi device driver for transmission. The packet structure and raw packet operation is depicted below:
```

Comment:
In this section it is probably good to mention that the MAC layer in nRF70 won't modify the contents, but the actual transmission of the packets will follow all typical MAC rules (e.g. link-layer acknowledgements, etc.)

Reply:
okay

Reply:
done